frontend/error.vue

```diff
@@ -0,0 +1,13 @@
+<script setup lang="ts">
+// We get the equivalent of props.error from useError() in the ModalError component.
+// const props = defineProps({
+//   error: Object
+// })
+
+const { error: { errorModalVisible } } = useModal()
+errorModalVisible.value = true
+</script>
+
+<template>
+  <ModalError :is-full-page="true" />
```

Comment:
IIRC, `error.vue` doesn't use the default layout. I'd consider asking it to using the approach I used earlier where you move layout.vue into `StandardLayout` so that it can be referenced by both this page + layout.vue, with the contents of the page going in as a `<slot/>`. The value add there would be that the RMI header + footer would still appear in the background, and it would be clearer to the user that they're experiencing an error on the page they were just on, rather than visiting a new site. The downside is that if we cause an error in the NavBar or Footer, we can't recover from it. That _probably_ doesn't matter though because our error-recovery strategies all rely on being able to go back to a page that does include the NavBar and footer (i.e. `router.back` or `router.push('/')`, so I don't think that's a strong reason not to do that.

Reply:
My idea was to keep this page as simple as possible for the reasons you mentioned: we don't know what caused the error or where it happened, so the fewer moving parts, the better.

Oddly enough though, halfway through my changes (I don't know what particularly), SSR errors went from having no layout (i.e. just `error.vue`), to having the full layout. I don't know how or why, but I'm going to leave this as is for now, as the behavior matches what you describe here.

Reply:
huh - a surprise but a welcome one. Thanks!